lmotel/templates/daemonset.yaml

```diff
@@ -0,0 +1,110 @@
+{{- if eq .Values.lm.logs_enabled true}}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "lmotel.fullname" . }}
+  labels:
+    app: {{ .Release.Name }}
+{{- if .Values.labels}}
+{{ toYaml .Values.labels| indent 4 }}
+{{- end }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        app: {{ .Release.Name }}
+{{- if .Values.labels}}
+{{ toYaml .Values.labels| indent 4 }}
+{{- end }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.logs_repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.arguments}}
+          args:  
+            {{ range .Values.arguments }}
+              - {{ . }}
+            {{ end }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: 80
```

Comment:
Port 80 should be removed because it is not required.  It is removed from statefulset.yaml with this PR [lmotel helm chart fixes and improvements](https://github.com/logicmonitor/k8s-helm-charts/pull/120)

Reply:
okay... removed the port from daemonset.